app/models/section.rb

```diff
@@ -152,7 +158,8 @@ def mark_as_exported!(exported_at = Time.zone.now)
   end
 
   def persisted?
-    editions.any?(&:persisted?)
+    previous_edition && previous_edition.persisted? ||
+      latest_edition.persisted?
```

Comment:
I've updated this so that I check `latest_edition.persisted?` first and I've wrapped the `previous_edition` conditions in parentheses.